tests/ui/specs/Scroller/ListOfThings/Scroller-GT-28384-specs.js

```diff
@@ -0,0 +1,48 @@
+const {expectFocusedItem} = require('../../VirtualList/VirtualList-utils');
+const ScrollerPage = require('../ScrollerPage');
+
+describe('Scroller List Of Things', function () {
+	beforeEach(function () {
+		ScrollerPage.open('ListOfThings');
+	});
+
+	it('should occur onScroll event [GT-28384]', function () {
+		// Precondition: Spotlight is on Close button. In this case, 'X' button is replaced 'Top' button.
+		ScrollerPage.buttonTop.moveTo();
+		ScrollerPage.spotlightSelect();
+		expect(ScrollerPage.buttonTop.isFocused()).to.be.true();
+
+		// Step3 -1: 5-way Down several times until reaching to the item which is under the screen.
+		const bottomVisibleIdNum = Number(ScrollerPage.bottomVisibleItemId().slice(4));
+		let index = 0;
+		for (; index < bottomVisibleIdNum + 1; index++) {
+			ScrollerPage.spotlightDown();
+			ScrollerPage.delay(100);
```

Comment:
It would be better to unify it in the same format as the below "for statement".
```suggestion
		let index = 0;
		for (; index <= bottomVisibleIdNum; index++) {
			ScrollerPage.spotlightDown();
			ScrollerPage.delay(100);
```